scripts/gitlab/check_line_width.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# check if line width of rust source files is not beyond x characters
+#
+
+
+BASE_BRANCH="origin/master"
+LINE_WIDTH="101"
```

Comment:
We should accept the maximum of 120, 100 characters is more the preferred number.

Reply:
Actually, the point of this check is exactly to notify people about exceeding the lower bound. 

Note, however, that this check doesn't block the merge it just acts as mere reminder to check if its possible to shorten the lines. 

Also, I'd say that if we set the limit to 120 here, we would effectively codify the 120 limit : )

Reply:
The problem is, that people are now already confused with the runtime check. They think that they need to make this happy and that they broke something. I think the same will happen with this one.

Reply:
maybe the text for these checks can be improved?

Reply:
Ah, yeah, @bkchr you are right. I believe the runtime check and this check and potentially future checks that don't fail the build are actually warnings/lints and should be displayed accordingly.
Unfortunately, GitHub doesn't support it.

However, I think we already are treating them as warnings, i.e. not all changes to the runtime bumps the version (e.g. adding a comment under the srml directory).

But then, if people do get confused, we can, as @gabreal already noted, update the message to explicitly mention that:

1. this is a warning.
1. the rule which was violated, i.e. maximum 100 and 120 as hard limit,
1. the link to the style guide,

And the best thing, it cannot harm (unlike to the version check) if people try to make their lines below 100 not 120, if they after all above still perceive this as hard error : )

Reply:
Yeah sounds reasonable @pepyakin.

Reply:
I like green ticks. There is nothing wrong with the occasional line being over 100 (and under 120). It is certainly not deserving of a red cross.